README.md

```diff
@@ -41,13 +41,15 @@ Txgh supports a significant number of configuration options, and you'll need to
 
 3. If you've configured Txgh to upload diffs, visit your project's page in Transifex and upload each of the files described in your tx config. Use Transifex's categories feature to add a category of `branch:heads/master` to each resource. These full resources will provide the base set of translations for your project.
 
-3. At this point, you're ready to deploy Txgh. There are a number of ways to do this, including using a host like AWS or Heroku. It's important your Txgh instance is publicly accessible over the Internet, because the next few steps involve setting up webhooks, which rely on being able to reach it.
+4. At this point, you're ready to deploy Txgh. There are a number of ways to do this, including using a host like AWS or Heroku. It's important your Txgh instance is publicly accessible over the Internet, because the next few steps involve setting up webhooks, which rely on being able to reach it.
```

Comment:
Any Markdown Ninjas know why the automatic numbering didn't work properly here? You should be able to number them all "1." and just work.

Reply:
Yeah, it will render properly, but the source will be still messed 🤷‍♂